src/CC-BY-NC-SA-2.0-UK.xml

```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-BY-NC-SA-2.0-UK" name="Creative Commons Attribution - Non-Commercial - Share-Alike 2.0 England and Wales">
```

Comment:
```suggestion
   <license isOsiApproved="false" licenseId="CC-BY-NC-SA-2.0-UK" name="Creative Commons Attribution Non Commercial Share Alike 2.0 England and Wales">
```
(the other Creative Commons licenses in the SPDX License List don't have dashes in the license name field)

Reply:
I just meant to highlight the difference between the titles in the legal code ("Attribution - Non-Commercial - Share-Alike 2.0 England and Wales" vs "Attribution-NonCommercial-ShareAlike 2.0"). I am relatively okay with changing that though, since it looks like it does not affect the text file or the `<titleText>` part of the XML file?

Reply:
> I am relatively okay with changing that though, since it looks like it does not affect the text file or the `<titleText>` part of the XML file?

Indeed, it doesn't affect either of those: the `<titleText>` element contains the name of the license as it's actually shown in the license text, whereas the `name` attribute of the `license` tag is used for the 'full name' column of the table on the [SPDX License List webpage](https://spdx.org/licenses/).

The text file is only used as part of the automated tests, and isn't published into the [license-list-data repository](https://github.com/spdx/license-list-data/) or the aforementioned webpage :)